src/app/services/user-store.service.ts

```diff
@@ -0,0 +1,50 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+
+@Injectable({ providedIn: 'root' })
+export class UserStoreService {
+  readonly userNameSubject = new BehaviorSubject<string>('Marcus Åström');
+  userName$ = this.userNameSubject.asObservable();
+
+  readonly placeOfWorkSubject = new BehaviorSubject<string>('IXI');
+  placeOfWork$ = this.placeOfWorkSubject.asObservable();
+
+  readonly notificationsSubject = new BehaviorSubject<
```

Comment:
i thikn we should leverage typesccript more here and provide a proper type to the Behaivour subject.

One example could be:

```
export const notificationTypes = {
    error: 'error',
    success: 'success',
    information: 'information',
    warning: 'warning'
  } as const

  export type NotificationType = (typeof notificationTypes)[keyof typeof notificationTypes]

  export interface Notification {
  notification: string,
  type: NotificationType
}
  
  readonly notificationsSubject = new BehaviorSubject<Notification[]>([
```

Reply:
Added a simple Notification type. Remember, this is a demo/test page for us to validate that our component works in Angular. I does not have to be perfect right now.

Reply:
Is the purpose of this demo application only internal testing? If so, then i agree but if other angular developers will have it as reference when they start migrating to tegel, then i think it should be typed properly even if it's a simple demo app

Reply:
Right now its only purpose is to validate that the components work in Angular.  In the future it would function as a demo application for angular developers, however before that it needs design input and so on.